src/annotations/AnnotationManager.php

```diff
@@ -135,6 +135,12 @@ class AnnotationManager
     private $_cacheSeed = '';
 
     /**
+     * Whether this version of PHP has support for traits.
+     */
+    private $_traitsSupported;
+
+
```

Comment:
Please remove extra empty line. There should be 1 empty line between class member (property/method) declarations.